article/app/controllers/LiveBlogController.scala

```diff
@@ -353,17 +367,20 @@ class LiveBlogController(
     filterKeyEvents.getOrElse(false)
   }
 
-  def getTopMentionsByTopics(blogId: String, topics: Option[String]) = {
-    val topMentionsResult = for {
-      topMentionTopic <- TopMentionsTopic.fromString(topics)
-      topMentions <- topMentionsService.getTopMentionsByTopic(blogId, topMentionTopic)
-    } yield topMentions
+  def getTopMentionsByTopicsAndPath(blogId: String, topics: Option[String], filterKeyEvent: Boolean) = {
+    if (filterKeyEvent) None
```

Comment:
Minor point but to make this a bit more functional I would actually move this if statement outside this function and only call the function if filterKeyEvents is false

Reply:
Thanks @alinaboghiu for pointing this out. I think the reason I did it this way was to more easily test it. But I now moved this condition out, and updated the test for it as well. Does it look better?